pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -249,6 +253,38 @@ var (
 	RotationPolicyAlways PrivateKeyRotationPolicy = "Always"
 )
 
+// OutputFormatType specifies which additional key formats should be added to Kubernetes secrets.
+// Allowed values are `DER` or `CombinedPEM`.
+// When Type is set to `DER` an additional entry `key.der` will be created in the secret
+// containing the binary format of the key.
+// When Type is set to `CombinedPEM` an additional entry `tls-combined.pem` will be created in the secret
+// containing the PEM formatted certificate chain and key (tls.key + tls.crt concatenated)
+// +kubebuilder:validation:Enum=DER;CombinedPEM
+type OutputFormatType string
+
+const (
+	// AdditionalKeyOutputFormatDER requests that the DER binary format of the key
+	// is stored in the `key.der` key of a secret
+	AdditionalKeyOutputFormatDER OutputFormatType = "DER"
+
+	// AdditionalKeyOutputFormatDERKey is the name of the data entry in the Secret resource
+	// used to store the DER formatted private key
+	AdditionalKeyOutputFormatDERKey string = "key.der"
```

Comment:
Same with prefix inconsistency. 

```suggestion
	AdditionalOutputFormatDERKey string = "key.der"
```